src/resolvers/auth.js

```diff
@@ -16,7 +16,10 @@ const {
 const createResetPasswordEmail = resetToken => {
   const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
   return {
-    from: EMAIL_FROM_ADDRESS,
+    from: {
```

Comment:
hmm might be worthwhile to extract this entire `from` as a const?

Reply:
yeah, i think so

Reply:
Fixed in a024aec